docs/excel/excel-add-ins-core-concepts.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Excel JavaScript API core concepts
+title: Beginner Excel add-in programming
 description: Use the Excel JavaScript API to build add-ins for Excel.
-ms.date: 12/04/2017
+ms.date: 10/03/2018
 ---
 
 
-# Excel JavaScript API core concepts
+# Beginner Excel JavaScript add-in programming
```

Comment:
Some years ago there was a session on designing developer documentation. One of the things I remember from it was to avoid using the word "Beginner" in titles. Apparently, a large percentage of developers, including beginners, will skip articles that use that word, out of ego. The recommendation was to use terms like "Basic" or "Fundamental" or "Essential" or "Foundational".

Reply:
Not that I remember.